src/EditorFeatures/Core/ExternalAccess/VSTypeScript/VSTypeScriptGoToDefinitionServiceFactory.cs

```diff
@@ -1,47 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@MariaSolOs TS/JS.  This file was just renamed to VSTypeScriptNavigableItemsServiceFactory